eos-split-flatpak-repo

```diff
@@ -114,47 +114,69 @@ def gather_remotes(repo):
     return os_remotes, flatpak_remotes
 
 
+def get_refspec_repo_type(refspec, os_remotes, flatpak_remotes):
+    """Determine the repo type this refspec should be a part of
+
+    Returns a RepoType or None if it can't be determined.
+    """
+    _, remote, ref = OSTree.parse_refspec(refspec)
+    if remote:
+        if remote in os_remotes:
+            return RepoType.OS
+        elif remote in flatpak_remotes:
+            return RepoType.Flatpak
```

Comment:
Is it possible for the remote to be in both `os_remotes` and `flatpak_remotes`?

Reply:
It shouldn't be the way they're constructed, but this function doesn't know that. I think an assert that they're disjoint sets would be useful. I'll add that.